src/SingleSelectOption.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import propTypes from 'prop-types'
+import cx from 'classnames'
+import { colors, spacers } from './theme.js'
+
+const SingleSelectOption = ({
+    value,
+    label,
+    active,
+    disabled,
+    onClick,
+    className,
+}) => (
+    <a
```

Comment:
I recall a discourse on Slack where the topic was when to use anchor elements. I _thought_ that the intent was to use anchor elements for navigation only. That's not a very pragmatic stance, so I do not criticize using anchors in this situation, as `a` is a little bit more semantic than a `span`.

However, I'm raising this to wonder if we could reasonably use a `datalist` + `option`[1] elements as they are widely supported in browser and would be semantically correct?

[1] https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist

Reply:
Interesting, I didn't know that element. I'll try that.

Reply:
See here: https://dhis2.slack.com/archives/CBM8LNEQM/p1572268672096300. After looking at it, it seems to me that its more intended for autocompletion.

> I took a look at using the datalist for wrapping the options, but I think it has a different intended use. From reading the mdn docs after your suggestion I thought that this'd be a nicely semantic wrapper for the options. But it's hidden by default.
>
> I found an article explaining that you can use the datalist to create an autocompleting input without js: https://jsfiddle.net/mqyh96uz (see https://www.jotform.com/blog/html5-datalists-what-you-need-to-know-78024/). I assume that that's its intended usecase? That would also explain why it's not visible by default. I had no idea that this was possible by the way.

Please correct me if I'm wrong though.